databases/week2/Week2HWTASK.sql

```diff
@@ -0,0 +1,56 @@
+-- Part 1: Working with tasks
+-- Add a task
+select * from task;
+INSERT INTO task 
+VALUES 
+(36,"Do MYSQL HYF exercises","Read the exercises and resolve it","2018-10-10 10:30:05","2018-10-09 10:30:05","2018-10-20 11:30:05",1);
+
+ -- Change the title of a task
+ALTER TABLE task RENAME COLUMN title TO task_title;
+UPDATE task set task_title = "Play music"  WHERE id=11;
+-- Change a task due date
+UPDATE task set due_date = "2017-11-20 23:58:03" WHERE id=2;
+-- Change a task status
+UPDATE task set status_id = 3 WHERE id=1;
+-- Mark a task as complete
+UPDATE task set status_id = 3 WHERE task_title="Charge electric bicycle" AND id=22;
+-- Delete a task
+DELETE FROM task where id=22;
+
+-- Part 2: School database
+-- Creating class table
+CREATE TABLE class(id INT PRIMARY KEY,name VARCHAR(50),begins DATE,ends DATE);
+-- Creating student table
+CREATE TABLE student(id INT PRIMARY KEY,name VARCHAR(50),email VARCHAR(320),phone VARCHAR(255),
+class_id INT,FOREIGN KEY (class_id) REFERENCES class(id));
+-- Create an index on the name column of the student table.
+CREATE INDEX idx_name ON student (name);
+-- Add a new column to the class table named status which can only have the following values: not-started, ongoing, finished
+ALTER TABLE class ADD status ENUM("not-started", "ongoing", "finished");
+
+-- Part 3: More queries
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.* FROM task 
```

Comment:
The `FROM` should be in a new line ;)

And it would be nice to see the user email to make sure the query is correct.

Reply:
Done :-)